### ISSUE TITLE
Build failure with llvm 10.x

### ISSUE THREAD

**Issue by david-geiger** [lightspark/lightspark#475]
Hi,

I tried on Mageia Cauldron to build lightspark against new llvm 10.0.0 but it fails to build:

[build.0.20200325064705.log](https://github.com/lightspark/lightspark/files/4379446/build.0.20200325064705.log)

Regards,
David

**Comment by dbluelle** [comment]
Yeah, currently building with -DENABLE_LLVM is disabled by default, because of the frequent changes in the llvm API and because the llvm based interpreter doesn't work anyway.
Building without -DENABLE_LLVM should work without problems.
